docs/operator-manual/ingress.md

```diff
@@ -414,6 +414,132 @@ Once we create this service, we can configure the Ingress to conditionally route
       - argocd.argoproj.io
 ```
 
+## [Istio](https://www.istio.io)
+You can put ArgoCD behind Istio using following configurations. Here we will achive both serving ArgoCD behind istio and using subpath on Istio
+
+First we need to make sure that we can run ArgoCD with subpath (ie /argocd). For this we have used install.yaml from argocd project as is
+
+```bash
+curl -kLs -o install.yaml https://raw.githubusercontent.com/argoproj/argo-cd/master/manifests/install.yaml
```

Comment:
I think it will be better point to stable version here, wdyt?

Reply:
Let me fix it.. You are correct..

Reply:
Done